uat/testing-features/src/main/java/com/aws/greengrass/steps/FileSteps.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package com.aws.greengrass;
+package com.aws.greengrass.steps;
```

Comment:
Moved the steps into the `steps` package because now we are adding in this utility classes. NetworkUtility